Lib/venv/scripts/nt/activate.bat

```diff
@@ -21,8 +21,7 @@ set PROMPT=__VENV_PROMPT__%PROMPT%
 if defined PYTHONHOME set _OLD_VIRTUAL_PYTHONHOME=%PYTHONHOME%
 set PYTHONHOME=
 
-if defined _OLD_VIRTUAL_PATH set PATH=%_OLD_VIRTUAL_PATH%
-else set _OLD_VIRTUAL_PATH=%PATH%
+if defined _OLD_VIRTUAL_PATH set PATH=%_OLD_VIRTUAL_PATH% else set _OLD_VIRTUAL_PATH=%PATH%
```

Comment:
This will never execute the `else` clause, and if `_OLD_VIRTUAL_PATH` is defined it will set `PATH` literally to the evaluation of the string `"%_OLD_VIRTUAL_PATH% else set _OLD_VIRTUAL_PATH=%PATH%"`.

Why again were parentheses stripped out of this script? Whatever the reason, there must be a better way. It's dysfunctional to not be able to write this statement as follows:

    if defined _OLD_VIRTUAL_PATH (
        set PATH=%_OLD_VIRTUAL_PATH%
    ) else (
        set _OLD_VIRTUAL_PATH=%PATH%
    )

Reply:
I didn't mention it previously, but in the current form the script errors out:

C:\Program Files\Python38\Lib\venv\scripts\nt>activate.bat
'else' is not recognized as an internal or external command,
operable program or batch file.

But good point on the parenthesis to make it function properly again.

Reply:
The parentheses were stripped out because stray parentheses and quotes were breaking the script for people.

Having `if defined ...` followed by `if not defined ...` on separate (single) lines is probably the best approach.